docs/overview/installation.md

```diff
@@ -82,6 +56,20 @@
     ]
     ```
 
+5. Add django-component's URL paths to your `urlpatterns`:
```

Comment:
Previously the installation step to add django-component's urlpatterns was in section for adding support for JS and CSS. Moved it up here into the "core" installation steps.

Back then it made sense for it to be up there.

But since we have the feature to automatically create endpoints for components with `View.public = True`, it means that we need to set `urlpatterns` not just when using JS / CSS, but all the time.